index.js

```diff
@@ -11,40 +11,37 @@ function instrument(options) {
   var instrumenter = new Instrumenter.Instrumenter();
   var captured = false;
 
-  function transform(file) {
-    // If if doesnt match the pattern dont instrument it
+  return through.obj(function(row, enc, next) {
+    if (row.file.indexOf("node_modules") !== -1) {
```

Comment:
I can see your point. In my projects I'm using a folder structure like this:

```
/myproject
|- /node_modules      <--- external modules from NPM
|-/src
   |-/node_modules    <--- project specific modules that are commited to github
   |-/pages           <--- from here I can require any module from both folders without using relative paths 
```
Therefore many times I need to instrument my modules inside `/src/node_modules` too - specially when I'm doing some integration tests. 

We could pass another param just to enable/disable this, but for consistency I still believe that making that option part of `excludePatterns` is the right way to go, although it introduces a breaking change.

What do you think?
Thanks
F